pkg/operation/botanist/component/kubeapiserver/secrets.go

```diff
@@ -330,24 +330,6 @@ func (k *kubeAPIServer) reconcileSecretHTTPProxy(ctx context.Context) (*corev1.S
 	return secret, nil
 }
 
-func (k *kubeAPIServer) reconcileSecretLegacyVPNSeed(ctx context.Context) (*corev1.Secret, error) {
-	if k.values.VPN.ReversedVPNEnabled {
-		return nil, nil
-	}
-
-	secret, err := k.secretsManager.Generate(ctx, &secretutils.CertificateSecretConfig{
-		Name:                        secretNameLegacyVPNSeed,
```

Comment:
Can we now drop https://github.com/gardener/gardener/blob/44246e18b204925bc8fe782ef978ab1cdfa4066c/pkg/operation/botanist/component/kubeapiserver/deployment.go#L47?